src/atomate2/forcefields/jobs.py

```diff
@@ -0,0 +1,81 @@
+"""Job to relax a structure using a force field (aka an interatomic potential)."""
+
+from __future__ import annotations
+
+import logging
+from dataclasses import dataclass, field
+
+from jobflow import Maker, job
+from pymatgen.core.structure import Structure
+
+from atomate2.forcefields.schemas import FFStructureRelaxDocument
+
+logger = logging.getLogger(__name__)
+
+
+@dataclass
+class CHGNetRelaxMaker(Maker):
+    """
+    CHGNet relax maker.
+
+    Parameters
+    ----------
+    name: str
+        The job name.
+    relax_cell : bool
+        Whether to allow the cell shape/volume to change during relaxation.
+    relax_kwargs : dict
+        Keyword arguments that will get passed to :obj:`StructOptimizer.relax`.
+    optimizer_kwargs : dict
+        Keyword arguments that will get passed to :obj:`StructOptimizer()`.
+    keep_info : list
+        Which information from the relaxation trajectory to save using
+        the :obj:`.FFStructureRelaxDocument.from_chgnet_result`.
+
+    """
+
+    name: str = "CHGNet relax"
+    relax_cell: bool = False
+    relax_kwargs: dict = field(default_factory=dict)
```

Comment:
What do you think about removing the relax_cell argument and just putting it in the default relax_kwargs. E.g.
```suggestion
    relax_kwargs: dict = field(default_factory={"relax_kwargs": False})
```

Reply:
I'm slightly on the fence about this, solely because I think it is the most important parameter that a user can set (hence I was drawing special attention to it by having it as a separate variable). Let me know your thoughts.

Reply:
Having thought about this more, I think we should definitely remove it. For the VASP makers, we don't have flags which control the input set parameters directly on the maker itself. All settings for controlling the actual calculation should be in optimizer_kwargs. To make things easier, I'd suggest we explicitly reproduce some of the defaults so that it is clear what calculation is happening and in case the chgnet defaults change.

E.g.,

```python
relax_kwargs: dict = field(default_factory={"relax_kwargs": False, "steps": 500, "fmax": 0.1})
```

And then we can outline what these specific parameters mean in the docstring and point the user to the chgnet documentation for more details.

Reply:
Actually, I think I changed my mind about this. The CHGNet relaxation is much simpler than the VASP calculations, so perhaps its ok to have a couple of parameters specified in the Maker. But I think you should also add a steps attribute too.

Reply:
Yep, exactly.

Reply:
Wait which one? I edited my original comment